config/purposes/bioscan.yml

```diff
@@ -0,0 +1,110 @@
+# Bioscan plate purposes
+---
+# Entry point plate created by sample manifest for insect in ethanol plates
+LILYS-96 Stock:
+  :asset_type: plate
+  :stock_plate: true
+  :cherrypickable_target: false
+  :input_plate: true
+  :presenter_class: Presenters::StockPlatePresenter
+  :creator_class: LabwareCreators::Uncreatable
+  :size: 96
+  # TODO: do we want any metadata entry?
+  # :custom_metadata_fields:
+  #   - metadatafieldname
+# LBSN-96 Lysate plate is the end result of lysis prep using the Beckman robot, or
+# it can be the start point if the Lysate plate was prepared manually
+# Once this plate is made we want to close off the lysate prep submission requests
+# Has a specific creator to add randomised controls
+LBSN-96 Lysate:
+  :asset_type: plate
+  :stock_plate: true
+  :cherrypickable_target: false
+  :input_plate: false
+  :presenter_class: Presenters::StockPlatePresenter
+  # TODO: once new labware creator is made add line for creator class
+  # :creator_class: LabwareCreators::<new bioscan creator>
+  :state_changer_class: StateChangers::AutomaticPlateStateChanger
+  :work_completion_request_type: 'limber_bioscan_lysate_prep'
+  :size: 96
+  # TODO: do we want any metadata entry?
+  # :custom_metadata_fields:
+  #   - metadatafieldname
+LBSN-384 PCR 1:
+  :asset_type: plate
+  :creator_class: LabwareCreators::QuadrantStamp
+  # TODO: is this needed? we don't split and re-merge later here
+  :merger_plate: true
+  # TODO: what presenter should this plate have?
+  # :presenter_class: Presenters::?
+  :label_template: plate_6mm_double
+  :size: 384
+  # TODO: do we want any metadata entry?
+  # :custom_metadata_fields:
+  #   - metadatafieldname
+LBSN-384 PCR 2:
+  :asset_type: plate
+  # TODO: May need a specific version of TaggedPlate creator to persist tag plate index 1-24
+  :creator_class: LabwareCreators::TaggedPlate
+  :presenter_class: Presenters::MinimalPcrPlatePresenter
+  :label_template: plate_6mm_double
+  :size: 384
+  # TODO check the names for this list of 24 tag layout templates (see tag plate gen stories)
+  :tag_layout_templates:
+    - Bioscan_384_template_1
```

Comment:
This is a problem if we want to introduce new versions. this set should be flagged v2, but any new set is going to require a change to this config and a re-deploy.

Reply:
Except there may be 2 sets in progress at a time, e.g. R&D are testing a new version whilst production is on the previous one. So after discussion we decided to build distinctly named tag groups and templates for each version.